lib/mnesia/src/mnesia.erl

```diff
@@ -2705,7 +2705,7 @@ install_fallback(Opaque) ->
 -spec install_fallback(Src::term(), Mod::module()|[Opt]) ->
           result() when
       Opt :: Module | Scope | Dir,
-      Module :: {'module', Mod::module()},
+      Module :: {'module', module()},
```

Comment:
@IngelaAndin is this change correct from your perspective? Notice how:
```
Mod::module()|[Opt].
```
but also below we were saying that
```
Mod :: module().
```
This PR makes the names of the type annotations globally bound to the function that defines them, i.e., the variable name is bound for that function and re-defining the same name with a different type (in that function) is an error

Reply:
@dgud I think should look at this.

Reply:
approved by @dgud via PM